easybuild/easyblocks/g/gromacs.py

```diff
@@ -444,21 +481,37 @@ def sanity_check_step(self):
 
         # in GROMACS v5.1, only 'gmx' binary is there
         # (only) in GROMACS v5.0, other binaries are symlinks to 'gmx'
+        # bins/libs that never have an _mpi suffix
         bins = []
         libnames = []
+        # bins/libs that may have an _mpi suffix
+        mpi_bins = []
+        mpi_libnames = []
         if LooseVersion(self.version) < LooseVersion('5.1'):
-            bins.extend(['editconf', 'g_lie', 'genbox', 'genconf', 'mdrun'])
+            mpi_bins.extend(['mdrun'])
 
         if LooseVersion(self.version) >= LooseVersion('5.0'):
-            bins.append('gmx')
-            libnames.append('gromacs')
-            if LooseVersion(self.version) < LooseVersion('5.1') and self.toolchain.options.get('usempi', None):
```

Comment:
Why was this removed?

Reply:
Because it's handled by the mpi_bins.extend(['mdrun']) above